### ISSUE TITLE
rock throws exception on non-existent classpath elements

### ISSUE THREAD

**Issue by fasterthanlime** [fasterthanlime/rock#908]
It should be a warning at most

**Comment by zhaihj** [comment]
It seems that there's no clean way to throw a warning in front-end?
Only middle-end can print a warning by `Resolver throwError(Warning)`, 
but the resolver is not available in front-end.

**Comment by fasterthanlime** [comment]
That's true, the current workaround used in various parts of rock is to go through the token:

```ooc
err := InternalError new(node token, "something")
node token module params errorHandler onError(err)
```

As for `AstBuilder`, you have direct access to `params`, so that should be even easier.

**Comment by fasterthanlime** [comment]
Closed in 784b3c31a155e415508ccbe861a37becb8a0a199
